api/services/ocean/spark/responses/appCompletionTrend.yaml

```diff
@@ -0,0 +1,17 @@
+description: number of completed, killed or failed apps per time range.
```

Comment:
Number of applications reaching a terminal state, grouped by the terminal state and a time bucket.